revscoring/languages/dutch.py

```diff
@@ -0,0 +1,177 @@
+import sys
+
+from .space_delimited import SpaceDelimited
+
+try:
+    from nltk.stem.snowball import SnowballStemmer
+    stemmer = SnowballStemmer("dutch")
+except ValueError:
+    raise ImportError("Could not load stemmer for {0}. ".format(__name__))
+
+try:
+    from nltk.corpus import stopwords as nltk_stopwords
+    stopwords = set(nltk_stopwords.words('dutch'))
+except LookupError:
+    raise ImportError("Could not load stopwords for {0}. ".format(__name__) +
+                      "You may need to install the nltk 'stopwords' " +
+                      "corpora.  See http://www.nltk.org/data.html")
+
+try:
+    import enchant
+    dictionary = enchant.Dict("nl")
+except enchant.errors.DictNotFoundError:
+    raise ImportError("No enchant-compatible dictionary found for 'nl'.  " +
+                      "Consider installing 'myspell-nl'.")
+
+
+badwords = [
+    r"aars",
+    r"an(aal|us)\w*",
+    r"balhaar",
+    r"drol(len)?",
+    r"fack(en|ing|s)?", "facking",
+    r"flikkers?",
+    r"focking",
+    r"ge(ile?|lul)",
+    r"geneukt",
+    r"hoer(en?)?",
+    r"homos?",
+    r"kaka?",
+    r"kak(hoofd|ken)",
+    r"k[ae]nker",
+    r"klootzak(ken)?",
+    r"klote",
+    r"kont(gat|je)?",
+    r"pedo",
+    r"penis(sen)?",
+    r"peop",
+    r"piemels?",
+    r"pijpen",
+    r"pik",
+    r"pimel",
+    r"pipi",
+    r"poep(chinees?|en|hoofd)?",
+    r"poep(ie|je|sex|te?)s?",
+    r"porno?",
+    r"neuke?",
+    r"neuken(de)?",
+    r"neukt(en?)?",
+    r"stron(d|t)",
+    r"suck(s|t)?",
+    r"zuigt",
+    r"sukkels?",
+    r"ter(ing|ten)", "tetten",
+    r"tieten",
+    r"vagina",
+    r"verekte",
+    r"verkracht",
+    r"dikzak",
+    r"dildo",
+    r"mon?g(olen|ool)?", "mooiboy",
+    r"negers?",
+    r"shit",
+    r"sperma",
+    r"kut(jes?)?",
+    r"stelletje",
+    r"losers?",
+    r"lul(len)?",
+    r"reet",
+    r"scheet", "scheten", r"schijt",
+    r"diaree",
+    r"slet",
+    r"lekkerding",
+    r"likken"
+]
+
+informals = [
+    r"aap(jes)?",
+    r"banaan",
+    r"bent",
+    r"boe(it)?",
+    r"doei"
+    r"dombo",
+    r"domme",
+    r"eigelijk",
+    r"godverdomme",
+    r"groetjes",
+    r"gwn",
+    r"hoi",
+    r"hal+o+",
+    r"heb",
+    r"hee+[jyl]",
+    r"houd?",
+    r"(hoi+)+",
+    r"hoor",
+    r"izan",
```

Comment:
I've never heard this word and can't seem to find any use of it in any Dutch material (except for a few rare uses of it as part of a name of something that isn't Dutch). It also doesn't sound or look like any existing word.